server/src/main/java/org/cloudfoundry/identity/uaa/audit/JdbcUnsuccessfulLoginCountingAuditService.java

```diff
@@ -40,20 +39,11 @@ public JdbcUnsuccessfulLoginCountingAuditService(
     @Override
     public void log(AuditEvent auditEvent, String zoneId) {
         switch (auditEvent.getType()) {
-            case MfaAuthenticationSuccess:
-                resetAuthenticationEvents(auditEvent, zoneId, MfaAuthenticationFailure);
-
-                break;
             case UserAuthenticationSuccess:
-            case PasswordChangeSuccess:
-                resetAuthenticationEvents(auditEvent, zoneId, UserAuthenticationFailure);
-                break;
-            case UserAccountUnlockedEvent:
+            case PasswordChangeSuccess, UserAccountUnlockedEvent:
```

Comment:
see sonar, move this into one list

https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&pullRequest=2717&id=cloudfoundry-identity-parent&open=AY2GPGZ4RapHNV97UMiS

Reply:
Cool, fixed.